src/twisted/web/wsgi.py

```diff
@@ -518,18 +522,26 @@ def wsgiError(started, type, value, traceback):
                     "WSGI application error",
                     failure=Failure(value, type, traceback)
                 )
+                if self._requestFinished:
+                    # Self.request.notifyFinish() has triggered, and
+                    # transport is potentially closed
+                    return
```

Comment:
Do we have a test dedicated to this branch?

Reply:
I added code which does another request where the response handler purposefully raises an exception. It hits this branch.